UserApi/UserApi/Controllers/UserController.cs

```diff
@@ -198,7 +198,21 @@ public async Task<IActionResult> GetJudges()
 
             return Ok(adJudges);
         }
-        
+        /// <summary>
+        ///     Remove Judge List Cache
+        /// </summary>
+        [HttpDelete("judges/cache", Name = "RemoveJudgeCache")]
+        [SwaggerOperation(OperationId = "RemoveJudgeCache")]
+        [ProducesResponseType(typeof(List<UserResponse>), (int)HttpStatusCode.NoContent)]
+        public async Task<IActionResult> RemoveJudgeCache()
+        {
+            const string KEY = "System.Func`1[System.Threading.Tasks.Task`1[System.Collections.Generic.IEnumerable`1[UserApi.Services.Models.UserResponse]]]";
```

Comment:
This doesn't look very readable. Why is the key look like an object's tostring method

Reply:
that's what @rehmanab set it too deliberately. @rehmanab is there a better way?

Reply:
I tried setting it to a const Key like "JUDGES_CACHE_KEY" on the Set, Get and Delete but it failed the tests

Reply:
Removed this bit.